editor.planx.uk/src/@planx/components/PlanningConstraints/Editor.tsx

```diff
@@ -33,34 +43,122 @@ function PlanningConstraintsComponent(props: Props) {
           title="Planning constraints"
           Icon={ICONS[TYPES.PlanningConstraints]}
         >
-          <InputRow>
-            <Input
-              format="large"
-              name="title"
-              placeholder={formik.values.title}
-              value={formik.values.title}
-              onChange={formik.handleChange}
-            />
-          </InputRow>
-          <InputRow>
-            <RichTextInput
-              name="description"
-              placeholder="Description"
-              value={formik.values.description}
-              onChange={formik.handleChange}
-            />
-          </InputRow>
-          <InputGroup label="Planning constraints data field">
+          <Typography variant="body2" sx={{ mb: 2 }}>
+            Find documentation about{" "}
+            <a
+              href="https://opensystemslab.notion.site/How-Planning-constraints-work-06b64035fe6b40279387375ef3de49e4?pvs=4"
+              target="_blank"
+            >
+              how constraints work in Plan✕ here
+            </a>
+            .
+          </Typography>
+          <InputGroup>
+            <InputRow>
+              <Input
+                format="large"
+                name="title"
+                placeholder={formik.values.title}
+                value={formik.values.title}
+                onChange={formik.handleChange}
+              />
+            </InputRow>
+            <InputRow>
+              <RichTextInput
+                name="description"
+                placeholder="Description"
+                value={formik.values.description}
+                onChange={formik.handleChange}
+              />
+            </InputRow>
             <InputRow>
               <Input
                 format="data"
                 name="fn"
                 placeholder={formik.values.fn}
                 value={formik.values.fn}
-                onChange={formik.handleChange}
+                disabled
```

Comment:
I think it's helpful to still show (rather than hide) this input so you know how this component will be recorded in your passport, but disabling it helps us ensure valid payloads later. If we're okay with this pattern, it's definitely one that would also be relevant & repeatable in FindProperty & DrawBoundary.